service-base/src/main/java/org/eclipse/hono/service/impl/RequestResponseEndpoint.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+package org.eclipse.hono.service.impl;
```

Comment:
I'd rather have this in the `org.eclipse.hono.service.amqp` package (same as `BaseEndpoint`).